scripts/mc_launch_rtp.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2021 The HERA Collaboration
+# Licensed under the 2-clause BSD License
+
+import os
+import sys
+import shutil
+import warnings
+import subprocess
+from hera_mc import mc
+from hera_mc.rtp import RTPLaunchRecord
+
+try:
+    import hera_opm.mf_tools as mt
+except ImportError:
+    sys.exit("hera_opm must be installed to use this script")
+
+ap = mc.get_mc_argument_parser()
+ap.description = """Launch an RTP workflow for the JD specified"""
+ap.add_argument("jd", type=int, default=None, help="JD to launch an RTP job for")
+ap.add_argument(
+    "workflow_config",
+    type=str,
+    default="/home/obs/src/hera_pipelines/pipelines/pipelines/h4c/rtp/v1/h4c_rtp_stage_1.toml",
+    help="hera_opm configuration to use for workflow",
+)
+ap.add_argument(
+    "working_directory",
+    type=str,
+    default="/home/obs/rtp_makeflow",
+    help="working directory for RTP",
+)
+ap.add_argument(
+    "--scan-files",
+    action="store_true",
+    default=False,
+    help=(
+        "Scan metadata of HERA data files before including in workflow. "
+        "Requires pyuvdata",
+    ),
+)
+ap.add_argument(
+    "--rename-bad-files",
+    action="store_true",
+    default=False,
+    help=(
+        "Rename files with bad metadata (found with --scan-files) with suffix "
+        "set by --bad-suffix.",
+    ),
+)
+ap.add_argument(
+    "--bad-suffix",
+    default=".METADATA_ERROR",
+    type=str,
+    help=(
+        "String to append to files pyuvdata could not read after running with "
+        "--scan-filea and --rename-bad-files. Default '.METADATA_ERROR'."
+    ),
+)
+ap.add_argument(
+    "conda_env",
+    default="RTP",
+    type=str,
+    help=(
+        "The conda environment to activate before launching RTP workflow."
+    ),
+)
+
+args = ap.parse_args()
+db = mc.connect_to_mc_db(args)
+
+if args.jd is None:
+    # launch a separate job for each un-launched day
+    jd_list = []
+    query = mc.query(RTPLaunchRecord).filter(RTPLaunchRecord.rtp_attempts == 0)
```

Comment:
do you want to use your getting methods here instead?

Reply:
Sure, I can do that. I had written this script before I wrote the `get_*` methods, so I'll use them here instead.

Reply:
This was done in ee9a244.